services/core-api/app/api/projects/information_requirements_table/resources/information_requirements_table.py

```diff
@@ -42,25 +52,22 @@ def get(self, project_guid, irt_guid):
     @requires_any_of([MINE_ADMIN, MINESPACE_PROPONENT, EDIT_INFORMATION_REQUIREMENTS_TABLE])
     @api.marshal_with(IRT_MODEL, code=200)
```

Comment:
in hindsight, it really doesn't make much sense to keep the @api.marshal_with(IRT_MODEL) and also the params. 🤦‍♀️